caddy/caddy.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ "$APP_DOMAIN" == "localhost" ]; then
+    export SITE_ADDRESS=":${LISTEN_HTTP_PORT}"
+elif [ "$SSL" == "true" ]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)